plugins/modules/vmware_guest.py

```diff
@@ -2683,10 +2684,9 @@ def configure_disks(self, vm_obj):
             if expected_disk_spec['filename']:
                 self.add_existing_vmdk(vm_obj, expected_disk_spec, diskspec, scsi_ctl)
                 continue
-            if vm_obj is None or self.params['template']:
-                # We are creating new VM or from Template
-                # Only create virtual device if not backed by vmdk in original template
-                if diskspec.device.backing.fileName == '':
+
+            # Only create virtual device if not backed by vmdk in original template
+            if diskspec.device.backing.fileName == '':
                     diskspec.fileOperation = vim.vm.device.VirtualDeviceSpec.FileOperation.create
```

Comment:
```suggestion
            if diskspec.device.backing.fileName == '':
                diskspec.fileOperation = vim.vm.device.VirtualDeviceSpec.FileOperation.create
```

@bardielle The error message says `E117: over-indented`.

Reply:
You're right, I missed the last failure (after re-running the recheck), sorry, and thank you.

Reply:
np